### PR TITLE
Update de.yaml

### PR DESCRIPTION
**Pull request by ninolus**
updated the german translations to be more natural

### CODE REVIEW

**Comment by shinuza** [comment]
Hey, thank you for your contribution? How did you edit/generate this file? It seems as if the entire thing was re-written, so it makes it hard for us (or people speaking the language) to figure out what changed.

**Comment by ninolus** [comment]
Hi I used BabelEdit, I am not sure what it did to the file, but i made a lot of changes.

**Comment by ninolus** [comment]
I will try to change the order so it is easier to see the changes I made

**Comment by ninolus** [comment]
Hope this helps!

File: src/locales/de.yaml [hunk 1]
```diff
@@ -7,40 +7,40 @@ app:
...
       history: Auftragsverlauf
-      config_files: Konfigurations Dateien
+      config_files: Konfigurationsdateien
     label:
-      accel_to_decel: Accel zu Decel
-      acceleration: Acceleration
-      sqv: Square Corner Velocity
-      velocity: Velocity
-      api_url: API URL
+      accel_to_decel: Beschleunigen zu Bremsen
```
Comment (FHeilmann) [inline src/locales/de.yaml:27]: Literal translation doesn't work too well here, maybe we can talk to Kevin what this parameter represents to find a better translation

File: src/locales/de.yaml [hunk 2]
```diff
@@ -7,40 +7,40 @@ app:
...
       printers: Drucker
       requested_speed: Gewünschte Geschwindigkeit
       services: Services
       speed: Geschwindigkeit
       flow: Flow
       z_offset: Z Offset
-      high: High
-      low: Low
+      high: Hoch
+      low: Niedrig
```
Comment (FHeilmann) [inline src/locales/de.yaml:43]: Are these limits? if so `Maximum` and `Minimum` would work better here

File: src/locales/de.yaml [hunk 3]
```diff
@@ -50,14 +50,14 @@ app:
       total_time_avg: Durchschnitt pro Druck
       total_print_time: Gesamte Druckzeit
       total_print_time_avg: Durchschnitt pro Druck
-      total_filament: Gesamt gebrauchtes Filament
+      total_filament: Gesamt verbrauchtes Filament
```
Comment (FHeilmann) [inline src/locales/de.yaml:53]: "Insgesamt verbrauchtes Filament"

File: src/locales/de.yaml [hunk 4]
```diff
@@ -118,53 +118,51 @@ app:
...
     label:
-      dirty: DIRTY
-      invalid: INVALID
+      dirty: VERUNREINIGT
+      invalid: UNZULÄSSIG
       up_to_date: AKTUELL
     status:
       updating: Aktualisieren...
-      finished: Aktualisieren beendet
+      finished: Aktualisieren abgeschlossen
```
Comment (FHeilmann) [inline src/locales/de.yaml:157]: Aktualisierung abgeschlossen

File: src/locales/de.yaml [hunk 5]
```diff
@@ -118,53 +118,51 @@ app:
...
     tooltip:
-      dirty: zeigt einen fehlerhaften gitrepo an
-      invalid: zeigt locale änderungen im gitrepo an
+      dirty: Deutet auf einen Fehler in Bezug auf das Git-Repsitory hin (z.B. detached head, not on master, invalid origin)
```
Comment (FHeilmann) [inline src/locales/de.yaml:159]: typo: Repsitory -> Repository

File: src/locales/de.yaml [hunk 6]
```diff
@@ -208,67 +205,70 @@ app:
...
     title:
       camera: Kamera | Kameras
-      macros: Macros
-      general: General
-      theme: Themen
-      tool: Tool
-      thermal_presets: Temperatur Presets
+      macros: Makros
+      general: Allgemein
+      theme: Aussehen
```
Comment (FHeilmann) [inline src/locales/de.yaml:210]: Suggestion: Farbschema

File: src/locales/de.yaml [hunk 7]
```diff
@@ -208,67 +205,70 @@ app:
...
+      tool: Druckkopf
+      thermal_presets: Temperatur-Presets
```
Comment (FHeilmann) [inline src/locales/de.yaml:212]: Temperatur-Voreinstellungen

File: src/locales/de.yaml [hunk 8]
```diff
@@ -208,67 +205,70 @@ app:
...
     label:
-      camera_url: Kamera Url
-      printer_name: Drucker Name
-      language: Anzeige Sprache
-      default_toolhead_move_length: Standardverfahrlänge des Tools
+      camera_fillspace: Platz füllen
```
Comment (FHeilmann) [inline src/locales/de.yaml:214]: Bereich füllen

File: src/locales/de.yaml [hunk 9]
```diff
@@ -208,67 +205,70 @@ app:
...
+      camera_fillspace_help: Den Platz der Kamera im Dashboard ausfüllen, wenn nur eine Kamera aktiv ist.
```
Comment (FHeilmann) [inline src/locales/de.yaml:215]: Platz -> Bereich

File: src/locales/de.yaml [hunk 10]
```diff
@@ -208,67 +205,70 @@ app:
...
+      time_estimates: Zeitschätzung
       invert_z_control: Z-Steuerung invertieren
       invert_y_control: Y-Steuerung invertieren
       invert_x_control: X-Steuerung invertieren
       enable: Aktivieren
       fps_target: FPS Ziel
     btn:
       add_camera: Kamera hinzufügen
-      select_theme: Thema auswählen
+      select_theme: Aussehen auswählen
```
Comment (FHeilmann) [inline src/locales/de.yaml:244]: Aussehen -> Farbschema

File: src/locales/de.yaml [hunk 11]
```diff
@@ -208,67 +205,70 @@ app:
...
       add_thermal_preset: Preset hinzufügen
       reset: Zurücksetzen
     timer_options:
       duration: Nur Dauer
       duration_description: Ähnlich wie bei einem Klipper-LCD wird hier nur die Dauer ohne Schätzungen angezeigt.
       file: Datei-Schätzung
       file_description: >-
-        Nimmt die Fortschrittsprozent und die Dauer, um die Gesamtdauer zu schätzen.<br />
-        Wird mit der Zeit genauer.
+        Schätzt die Gesamtdauer anhand des Fortschritts in Prozent und der verstrichenen Zeit.
```
Comment (FHeilmann) [inline src/locales/de.yaml:252]: Fortschritts -> Datei-Fortschritts

File: src/locales/de.yaml [hunk 12]
```diff
@@ -208,67 +205,70 @@ app:
...
+        <br /> Wird mit der Zeit genauer.
       filament: Filament-Schätzung
       filament_description: >-
-        Nimmt das verwendetes Filament vs. das geschätztes Filament, um die Gesamtdauer zu schätzen.<br />
-        Ist mit der Zeit genauer.
+        Schätzt die Gesamtdauer anhand der Länge des verwendeten Filaments im Vergleich zur Gesamtlänge.
```
Comment (FHeilmann) [inline src/locales/de.yaml:256]: verwendeten -> gedruckten 
or 
verwendeten -> verbrauchten

File: src/locales/de.yaml [hunk 13]
```diff
@@ -283,49 +283,49 @@ app:
     label:
       dir_name: Ordnername
       file_name: Dateiname
-      disk_usage: Benutzter Sepicher
+      disk_usage: Belegter Sepicher
```
Comment (FHeilmann) [inline src/locales/de.yaml:286]: Sepicher -> Speicher (typo)

File: src/locales/de.yaml [hunk 14]
```diff
@@ -283,49 +283,49 @@ app:
...
     msg:
       not_found: Keine Dateien gefunden
       uploading: Hochladen...
-      processing: Verarbeitung...
-      confirm: Bist du sicher? Dadurch werden alle darin enthaltenen Dateien und Ordner gelöscht.
+      processing: Verarbeiten...
+      confirm: Sind Sie sicher? Es werden alle enthaltenen Dateien und Ordner gelöscht.
     tooltip:
-      disabled: Aufträge werden vor der ersten Kommunikation mit klippy deaktiviert
+      disabled: Aufträge sind vor der ersten Kommunikation mit klippy deaktiviert
```
Comment (FHeilmann) [inline src/locales/de.yaml:293]: Aufträge -> Druckvorgänge

File: src/locales/de.yaml [hunk 15]
```diff
@@ -283,49 +283,49 @@ app:
...
     msg:
       trouble: >-
-        Haben Sie Probleme? <a href="%{url}" target="_blank">Siehe hier</a> für
-        mehr Informationen.
+        Haben Sie Schwierigkeiten? <a href="%{url}" target="_blank">Siehe hier</a> für
+        weitere Informationen.
     tooltip:
       endpoint_examples: >-
-        Geben Sie Ihre API-URL ein.<br />Ein paar Beispiele könnten sein:<br
+        Geben Sie Ihre API-URL ein.<br />Einige Beispiele könnten sein:<br
```
Comment (FHeilmann) [inline src/locales/de.yaml:321]: Einige Beispiele könnten sein -> Zum Beispiel

File: src/locales/de.yaml [hunk 16]
```diff
@@ -283,49 +283,49 @@ app:
...
         /><blockquote>http://fluidd.local, http://192.168.1.150</blockquote>
     error:
       cant_connect: >-
-        Irgendetwas ist schief gelaufen, und Fluidd kann das Ziel nicht erreichen. Sind Sie
+        Irgendetwas ist schief gelaufen und Fluidd kann das Ziel nicht erreichen. Sind Sie
```
Comment (FHeilmann) [inline src/locales/de.yaml:325]: Etwas ist schief gelaufen und Fluidd kann die Adresse nicht erreichen

**Review by FHeilmann — COMMENTED** [review]
Some comments from another native speaker.

File: src/locales/de.yaml (hunk 2 above)
Comment (cadriel) [reply src/locales/de.yaml:43]: They aren't limits, high and low are used where you have a sensor to show the known high and low values. Ie., rolling over the pi cpu temp sensor would give you the current, high and low temps.

**Comment by cadriel** [comment]
@ninolus Have you gone through the suggested changes? I'd like to get this merged! :)

File: src/locales/de.yaml (hunk 1 above)
Comment (ninolus) [reply src/locales/de.yaml:27]: I am still not sure how to translate this properly. Any help would be appreciated.

File: src/locales/de.yaml (hunk 2 above)
Comment (ninolus) [reply src/locales/de.yaml]: I changed it to translate to "highest" and "lowest". I hope that fits.

**Comment by ninolus** [comment]
@cadriel Sorry I didn't get around to take a look at this sooner. I am still struggling with translating "Accel to Deccel". Everything else should be fine now.

**Comment by cadriel** [comment]
@ninolus Ok, thanks a bunch. I think all of the changes are otherwise an improvement, so merging. We can always improve moving forward too!

**Pull request merged**